files/crond.pam

```diff
@@ -0,0 +1,10 @@
+#
+# The PAM configuration file for the cron daemon
+#
+#
+# No PAM authentication called, auth modules not needed
+account    required   pam_access.so
+account    include    password-auth
+#session    required   pam_loginuid.so
```

Comment:
can you make this optional instead, per https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=762157 and https://github.com/docker/docker/issues/5663